src/main/java/dota/buff/config/SpringServiceConfig.java

```diff
@@ -0,0 +1,23 @@
+package dota.buff.config;
+
+import com.github.wannesvr.core.Dota2ApiClient;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.PropertySource;
+
+@Configuration
+@ComponentScan("dota.buff.service")
```

Comment:
using spring boot you don't need to say manually which packages should be scanned

Reply:
Done